src/frontend/src/app/core/layout/primary-navbar/primary-navbar.component.ts

```diff
@@ -22,98 +22,100 @@ import NAVBAR_ITEMS, { NavbarDropdown, NavbarItem } from './navbar-items';
   ],
   template: `
     <header class="navbar">
-      <nav class="navbar-nav">
-        <a href="/" class="navbar-brand">
-          <img
-            class="navbar-logo"
-            src="assets/codegov-logo.png"
-            alt="CodeGov Logo"
-          />
-
-          <span class="navbar-company">codegov.org</span>
-        </a>
-
-        <div class="navbar-desktop-nav">
-          @for (item of navbarItems; track item.title) {
-            @if (isParentItem(item)) {
-              <app-dropdown menuTriggerClassName="navbar-nav-item">
-                <ng-container ngProjectAs="[menuTrigger]">
-                  {{ item.title }}
-                </ng-container>
-
-                <ng-container ngProjectAs="[menu]">
-                  @for (subItem of item.children; track subItem.title) {
-                    <a [href]="subItem.href" class="dropdown-item">
-                      {{ subItem.title }}
-                    </a>
-                  }
-                </ng-container>
-              </app-dropdown>
-            } @else {
-              <a [href]="item.href" class="navbar-nav-item">
-                {{ item.title }}
-              </a>
-            }
-          }
-        </div>
-
-        <div class="navbar-mobile-nav-trigger">
-          <button type="button" (click)="onSidenavOpenClicked()">
-            <span class="sr-only">Open main menu</span>
-            <app-hamburger-menu-icon />
-          </button>
-        </div>
-      </nav>
-
-      <div
-        class="navbar-mobile-nav"
-        [ngClass]="{ hidden: !isSidenavOpen }"
-        role="dialog"
-        aria-modal="true"
-      >
-        <div class="sidenav-backdrop" (click)="onSidenavCloseClicked()"></div>
-
-        <div class="sidenav">
-          <div class="sidenav-header">
-            <a href="/" class="sidenav-brand">
-              <img
-                class="sidenav-logo"
-                src="assets/codegov-logo.png"
-                alt="CodeGov Logo"
-              />
-
-              <span class="sidenav-company">codegov.org</span>
-            </a>
-
-            <button type="button" (click)="onSidenavCloseClicked()">
-              <span class="sr-only">Close menu</span>
-              <app-menu-close-icon />
-            </button>
-          </div>
-
-          <nav class="sidebar-nav">
+      <div class="container mx-auto">
```

Comment:
This diff looks shit unless you enable the "hide whitespace" option in the diff view settings.